src/components/Activity/Swipeout.js

```diff
@@ -180,6 +179,8 @@ var Swipeout = React.createClass({
       }
     }
 
+    this.props.onSwipeFinish(openRight);
+
```

Comment:
Since we decided to bring in the entire library file we should apply the ES6 rules.